src/components/ProjectCard/index.jsx

```diff
@@ -49,30 +50,49 @@ export default class ProjectCard extends Component {
     }
   };
 
+  handleCardClick = event => {
+    const {
+      project: { fileName },
+    } = this.props;
+
+    if (event.type === 'keypress') {
+      const code = event.charCode || event.keyCode;
+
+      if (code !== 32 && code !== 13) {
+        return;
+      }
+    }
+
+    this.props.history.push(`/projects/${fileName}`);
+  };
+
   render() {
     const {
       classes,
-      project: { name, summary, fileName },
+      project: { name, summary },
     } = this.props;
 
     return (
-      <NavLink className={classes.navlink} to={`/projects/${fileName}`}>
```

Comment:
Let's use `Link` instead of `NavLink`.

```jsx
import { Link } from 'react-router-dom';`
```

Reply:
@helfi92 but aren't both will render similarly as <a>?

Reply:
`NavLink` is meant to be used for navigation. When the URL matches the `to` prop, it will attach an`active` class to the component which we don't want to do here. `NavLink` uses `Link` internally.

Reply:
I see. In that case, you can do `component="object"`. When an element is wrapped with `<object>` the DOM will consider this as an external resource and will be treated without looking at the parent element. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/object.

```jsx
<Typography component="object" ...>
   ...
</Typography>
```

Reply:
WOW how did you find such simple solution ✨